R/install.R

```diff
@@ -0,0 +1,264 @@
+#' Toolchest Installation
+#'
+#' Installs and loads the `toolchest_client` Python package.
+#'
+#' @note The Python package interface is stored in the global variable `toolchest_client`.
+#'
+#' @export
+install_toolchest <- function() {
```

Comment:
Is it correct to say that the user has to run `install_toolchest`? The rest of the comment is assuming the steps to run for the R client right now are:

```
install.packages("devtools")
devtools::install_github("trytoolchest/toolchest-client-r")
library("toolchest")

toolchest::install_toolchest()

Sys.setenv(TOOLCHEST_KEY = "xxx")
toolchest::test(input_path = "./input.fastq", output_path = "./output.txt")
```

--

This is a clear improvement from it not working, but we're still requiring a call to `toolchest::install_toolchest()` after running what I think most users will assume is an install process. Is there a way we can make this happen automatically?

I can think of two processes that might work, but I lack full context:

1. Running `install_toolchest` on `.onLoad`
2. Running `install_toolchest` when running a tool (e.g. `kraken2`), and checking to see if it has been run yet